src/tequila/wavefunction/qubit_wavefunction.py

```diff
@@ -365,15 +365,18 @@ def normalize(self, inplace: bool = False) -> QubitWaveFunction:
     # because the __mul__ implementation of the number tries to perform some sort of array
     # operation.
     def length(self):
-        return sum(1 for (k, v) in self.raw_items() if abs(v) > 1e-6)
+        return sum(1 for _ in self.raw_items())
```

Comment:
Is this change on purpose? I'm guessing you branched off before https://github.com/tequilahub/tequila/pull/375?

Reply:
I did it but I overwrote it by mistake. Should I make a new one

Reply:
A new commit? Yes, or amend this one to remove the change. (Unless I misunderstood and you mean the change was on purpose?)

Reply:
No, it wasn't on purpose. I made a new commit.